scripts/generate_schema_docs/template.go

```diff
@@ -24,23 +24,9 @@ var tmpl = template.Must(template.New("").Parse(`# {{.Title}}
 <!-- These files are auto generated -->
 
 ## Properties
+| Property 	| Description 						| Type 	   			   | Required			   		   | Default             | Pattern 			 			 |
+| --------- | ----------------------- | ---------------- | --------------------- | ------------------- | ------------------- |
 {{- range $name, $_ := .Properties}}
-
-### {{$name}}
-
-{{- if .Description}}
-
-{{.Description}}
-{{- end}}
-
-{{- if .Type}}
-
-Type: {{.Type}}
-{{- end}}
-
-{{- if .Pattern}}
-
-Pattern: {{.Pattern}}
-{{- end}}
+| {{$name}} | {{or .Description "-"}} | {{or .Type "-"}} | {{.RequiredByParent}} | {{or .Default "-"}} | {{or .Pattern "-"}} |
```

Comment:
Should the fourth column be `{{or .RequiredByParent .Required}}`?

Reply:
I don't think so. The `Required` field is at parent level and it's an array of keys. Is converted to boolean flags for their properties while executing `generate_schema_docs`. If it's in `Required, it should be `RequiredByParent` at the next level